src/main/java/org/eclipse/tractusx/ssi/lib/crypt/x25519/x25519PrivateKey.java

```diff
@@ -60,7 +60,7 @@ public x25519PrivateKey(byte[] privateKey) throws InvalidPrivateKeyFormatExcepti
    *
    * @param privateKey the private key
    * @param pemFormat the pem format
-   * @throws InvalidePrivateKeyFormat the invalide private key format
+   * @throws InvalidPrivateKeyFormatException the invalide private key format
    * @throws IOException the io exception
```

Comment:
`IOException` is not thrown any more.

Reply:
yeah havn't checked these files, it was due to merge with the latest main will take it out